src/rules/useStrictRule.ts

```diff
@@ -34,61 +35,51 @@ class UseStrictWalker extends Lint.ScopeAwareRuleWalker<{}> {
         return {};
     }
 
-    public visitModuleDeclaration(node: TypeScript.ModuleDeclarationSyntax): void {
-        var childCount = TypeScript.childCount(node.modifiers);
+    public visitModuleDeclaration(node: ts.ModuleDeclaration) {
+        var modifiers = node.modifiers;
+        var hasDeclareModifier = (modifiers != null) && (modifiers.length > 0) && (modifiers[0].kind === ts.SyntaxKind.DeclareKeyword);
 
         // current depth is 2: global scope and the scope created by this module
-        // but skip declare module statements
-        if (this.getCurrentDepth() === 2 &&
-            !(childCount > 0 && TypeScript.childAt(node.modifiers, 0).kind() === TypeScript.SyntaxKind.DeclareKeyword)) {
-
-            if (this.hasOption(UseStrictWalker.OPTION_CHECK_MODULE)) {
-                this.checkUseStrict(node, node.moduleElements);
+        if (this.getCurrentDepth() === 2 && !hasDeclareModifier) {
+            if (node.body != null && this.hasOption(UseStrictWalker.OPTION_CHECK_MODULE)) {
+                this.handleBlock(node, <ts.Block> node.body);
```

Comment:
How do you know this isn't a ModuleDeclaration here? (Is it because of the declaration modifier? If so, that's fine, I'm just confused/curious. Also, minor detail -- this should probably be ModuleBlock and not Block, to match up with how body is typed.